tex/EqualX/Portfile

```diff
@@ -47,7 +47,7 @@ post-patch {
 }
 
 destroot {
-    copy ${worksrcpath}/${name}.app ${destroot}${applications_dir}
+    move ${worksrcpath}/equalx.app ${destroot}${applications_dir}/${name}.app
```

Comment:
why not just installing it with the app name that upstream decided on? Now you're changing it back to the capilized version, that seems just confusing to me...

Reply:
This will not require a revbump, since it worked correctly on case-insensitive OSs. I mean, if you insist, I will change, no problem, but then revbumping it.

Reply:
@reneeotten So should I change the name instead and revbump? So that we can merge the PR.

Reply:
that would be my preferences indeed

Reply:
Sure, no issues. Let me rebase.